pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -359,7 +359,7 @@ def apply(self):
 
     @async_action_nowait
     def execute(self):
-        """"Apply a partitioning."""
```

Comment:
The next method below has 4 "s too :)

Reply:
Fixed in https://github.com/rhinstaller/anaconda/pull/2364.